Assets/Plugins/Krearthur/GOPainter/Scripts/GOPainter.cs

```diff
@@ -386,35 +365,10 @@ protected void OnEnable()
             }
         }
 
-        protected void LoadAssets()
-        {
-            string basePath = "Assets/Krearthur/GOPainter/Resources/";
-
-            cursor = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint.png");
-            cursorDelete = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint_delete.png");
-            cursorLine = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint_line.png");
-            cursorLineDelete = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint_line_delete.png");
-            cursorRect = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint_rect.png");
-            cursorRectFill = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint_rect_fill.png");
-            cursorRectDelete = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint_rect_delete.png");
-            cursorCircle = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint_circle.png");
-            cursorCircleFill = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint_circle_fill.png");
-            cursorCircleDelete = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint_circle_delete.png");
-            cursorPick = AssetDatabase.LoadAssetAtPath<Texture2D>(basePath + "paint_pick.png");
-
-            canvasMaterial = AssetDatabase.LoadAssetAtPath<Material>(basePath + "PaintCanvasMat.mat");
-            brushMaterial = AssetDatabase.LoadAssetAtPath<Material>(basePath + "PaintBrushMat.mat");
-
-            brushPrefab = AssetDatabase.LoadAssetAtPath<GameObject>(basePath + "BrushRadius.prefab");
-            placeSound = AssetDatabase.LoadAssetAtPath<AudioClip>(basePath + "blop.ogg");
-        }
-
         protected void OnDisable()
         {
             if (!Application.isPlaying)
-            {
```

Comment:
Hi Valery,
90% of your changes are removing curly braces for one-line statements like this. 
But it is my personal preference to **almost always** use curly braces, because it is more readable for me.
Also when in the future some thing is added in the if statement, we dont have to add curly braces afterwards because they are always there.

Reply:
I agree. We'll add brackets, let's get to work!

Reply:
Brought the code to one common style, added parentheses for single-line positions (with the exception of `if ( ) return;` and `if ( ) continue;`), checked the rest of the scripts in which I had not previously made changes.